test/extended/openstack/volumes.go

```diff
@@ -280,6 +352,17 @@ func getMonitoringPvcs(ctx context.Context, dc dynamic.Interface) ([]objx.Map, e
 	return objects(objx.Map(obj.UnstructuredContent()).Get("items")), nil
 }
 
+// return list of PVCs defined in a given namespace
+func GetPvcsFromNamespace(ctx context.Context, dc dynamic.Interface, namespace string) ([]objx.Map, error) {
```

Comment:
What do you think of putting it in utils?

Reply:
For the moment it's only used in volumes.go so I think it's ok to keep it there.